frontend/src/pages/DetailedCourse/components/BackButton/BackButton.tsx

```diff
@@ -5,9 +5,9 @@ import { ButtonLabels } from 'constants/ButtonLabels';
 import { PAGES } from 'constants/pages';
 import { PATHS } from 'constants/routes';
 import { backIconMobile } from 'icons';
-import { IBackButton } from 'types/detailedCourse';
 
 import { BackArrow, BackWrapper, StyledButton } from './styled';
+import { IBackButton } from '../../types';
```

Comment:
maybe absolute path will be better here?

Reply:
done